cloudant-sync-datastore-android/src/main/java/com/cloudant/sync/sqlite/android/AndroidSQLite.java

```diff
@@ -109,7 +109,7 @@ public Cursor rawQuery(String sql, String[] values) {
 
```

Comment:
I believe the `update()` method should also have the `table` name quoted at line 102.